cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -18,29 +19,37 @@ import { defaultDeserializeUser, defaultSerializeUser } from './UserSerializer';
  */
 export class PassportAuth implements EndpointSecurity {
   protected passport: passport.Passport = new passport.Passport();
-  protected loginRoute: string;
+  protected jwtOpts: JwtOptions;
 
   // tslint:disable-next-line:max-line-length
-  constructor(protected readonly userRepo: UserRepository, protected readonly userService: UserService, loginRoute?: string) {
-    this.loginRoute = loginRoute || '/login';
+  constructor(protected readonly userRepo: UserRepository, protected readonly userService: UserService) {
   }
 
   /**
    * Initializes an Express application to use passport and express-session
+   * Note: If the session options is not defined, session will not be used.
    *
    * @param app - An express application
    * @param sessionOpts - Session options to be used by express-session
+   * @param secret - secret used for Passport's JWT strategy
    */
-  public init(app: express.Router, sessionOpts?: SessionOptions) {
+  public init(app: express.Router, sessionOpts?: SessionOptions, secret?: string) {
```

Comment:
`secret?: any` may be better => That's restrictive as people may pass also key (which is probably better to secure this)